index.html

```diff
@@ -74,6 +75,220 @@ <h1 class="main-text">I’m Giovanni Glad to see you!</h1>
+                </div>      
+            </div>
+
+            <div class="about-social-container">
+                <div class="about-connect-title">LET'S CONNECT</div>
+                    <div>
+                        <ul class="connect-icons">
+                            <li>
+                                <a href="">
+                                    <img class="twitter" src="./Assets/twitter.svg" alt="Connect Twitter">
+                                </a>
+                            </li>
+                            <li>
+                                <a href="">
+                                    <img class="Linkedin" src="./Assets/Linkedin icon.svg" alt="Connect Linkedin">
+                                </a>
+                            </li>
+                            <li>
+                                <a href="">
+                                    <img class="medium" src="./Assets/medium.svg" alt="Connect Medium">
+                                </a>
+                            </li>
+                            <li>
+                                <a href="">
+                                    <img class="github" src="./Assets/github.svg" alt="Connect Github">
+                                </a>
+                            </li>
+                            <li>
+                                <a href="">
+                                    <img class="angellist" src="./Assets/angellist.svg" alt="Connect Angel List">
+                                </a>
+                            </li>
+                        </ul>
+                    </div>
+                </div>
+
+                <div class="button-resume-div">
+                    <button class="button-resume">Get my Resume</button>
+                </div>
+
+                <div class="languages">
+                    <div class="language-title">
+                        <p>Languages</p>
+                    </div>
+                    <div class="languages-list">
+                        <ul>
+                            <li class="languages-li">
+                                <img src="/Assets/javascript_circle.png" alt="">
+                                <p>JAVASCRIPT</p>
+                            </li>
+                            <li class="languages-li">
+                                <img src="/Assets/html_circle.png" alt="">
+                                <p>HTML</p>
+                            </li>
+                            <li class="languages-li">
+                                <img src="/Assets/css_circle.png" alt="">
+                                <p>CSS</p>
+                            </li>
+                        </ul>
+                    </div>
```

Comment:
### Your Languages Section
![Capture](https://user-images.githubusercontent.com/93039359/198690803-995d27b2-6e62-4b99-9e54-3ebc98538599.PNG)

### Figma Languages Section
![Capture1](https://user-images.githubusercontent.com/93039359/198690812-f520d1ce-6037-4067-ab3b-93bcf2063725.PNG)

- [ ] Your language section is good but i cannot see any image as you can see in the Figma template. Kindly add these missing images to make your design look similar to the Figma template 👍